### PR TITLE
Add portfolio for Candice

### PR DESCRIPTION
**Pull request by candyhy**
Update contributions in candyhy.md
Fix typo in DG

### CODE REVIEW

File: docs/team/candyhy.md [hunk 1]
```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
...
+
+* **Enhancement**: Maintains unique `Property` and `Appointment` lists, and all their attributes.
+    * What it does: Supports other commands like `add`, `edit`, `sort`, `filter` as all property and appointment data are maintained in a list. The unique property and appointment lists prevents addition/ edit of `Property` / `Appointment` that results in duplicate  `Property` / `Appointment` listings.
+    * Justification: The core functionality of our application is heavily reliant on both of these lists, as `Property` and `Appointment` lists are necessary to maintain `Property` and `Appointment` data PocketEstate manages.
+    * Highlights: Prevents addition of duplicate `Property` and `Appointment` to minimise redundant data.
+
+<div style="page-break-after: always;"></div>
+
+* **Enhancement**: Implemented storage support for both `Property`, `Appointment` and their attributes.
+    * What it does: Performs accurate parsing of `Property` and `Appointment` data from the respective `Property` and `Appointment` lists to data form that is reader friendly for display to the user, which is then stored in separate json files.
```
Comment (dvdweien) [inline docs/team/candyhy.md]: would "lists to a data form that is reader friendly" be better?

**Review by dvdweien — COMMENTED** [review]
just a small suggestion

File: docs/team/candyhy.md [hunk 2]
```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
...
+
+#### New Features & Enhancements
+
+* **Enhancement**: Graphical User Interface (GUI)
+    * What it does: Simultaneously displays both property and appointment lists side by side
+    * Justification: This feature improves the product significantly because it enables integration of commands on both property and appointment lists, yet enables the user to conduct distinct commands on both lists and view its effects simultaneously without having to call `list property` or `list appointment` prior to each operation
+    * Highlights: Greying out of properties and appointments with expired deadlines in the GUI, a newly designed Light theme colour scheme with a organised, minimalist design that highlights important details.
+    The implementation was challenging due to limited experience and precise nature of fxml.
+
+<div style="page-break-after: always;"></div>
```
Comment (w2vgd) [inline docs/team/candyhy.md]: This line is used for page breaks when printing to pdf. The maximum page length of ppp is 2 pages, so you can use at most one time only

File: docs/team/candyhy.md [hunk 3]
```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
...
+    * Justification: The core functionality of our application is heavily reliant on both of these lists, as `Property` and `Appointment` lists are necessary to maintain `Property` and `Appointment` data PocketEstate manages.
+    * Highlights: Prevents addition of duplicate `Property` and `Appointment` to minimise redundant data.
+
+<div style="page-break-after: always;"></div>
+
+* **Enhancement**: Implemented storage support for both `Property`, `Appointment` and their attributes.
+    * What it does: Performs accurate parsing of `Property` and `Appointment` data from the respective `Property` and `Appointment` lists to data form that is reader friendly for display to the user, which is then stored in separate json files.
+    * Justification: Proper parsing and storage of `Property` and `Appointment` data ensures data integrity and reliability of PocketEstate that aims to manage significant property and appointment data. 
+    * Highlights: Maintains 2 separate json files with reader friendly data (Eg. Parsing of date from 01-12-2021 to 12 Dec 2021, time from 1500 to 3:00pm) to be presented to the user in GUI
```
Comment (w2vgd) [inline docs/team/candyhy.md]: The date is actually parsed to `Dec 12, 2021`

File: docs/team/candyhy.md [hunk 4]
```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
...
+    * Added user stories and use cases for list command. ([\#314](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/314), [\#38](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/38))
+    * Updated Storage component and Storage class diagram ([\#199](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/199))
+    * Formatted Appendix with numbering for each section ([\#322](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/322))
+
+#### Community
+
+* PRs reviewed (with non-trivial review comments): ([\#162](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/162), [\#126](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/126), [\#298](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/298), [\#111](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/111))
+* Provided tips through forum discussions (examples: [Configuring IntelliJ to follow module coding standards for switch statements](https://github.com/nus-cs2103-AY2021S2/forum/issues/42))
+* Provided help through forum discussions (examples: [\#23](https://github.com/nus-cs2103-AY2021S2/forum/issues/23))
+* Fixed a bug in the original AB3 Storage class diagram (issue: [\#273](https://github.com/nus-cs2103-AY2021S2/forum/issues/273))
```
Comment (w2vgd) [inline docs/team/candyhy.md]: I think need to rephrase this because you raised the bug but didn't fix it right?

**Review by w2vgd — COMMENTED** [review]
Some comments

File: docs/team/candyhy.md (hunk 2 above)
Comment (candyhy) [reply docs/team/candyhy.md]: oh okay will correct this

File: docs/team/candyhy.md (hunk 3 above)
Comment (candyhy) [reply docs/team/candyhy.md]: thanks for catching this!

File: docs/team/candyhy.md (hunk 4 above)
Comment (candyhy) [reply docs/team/candyhy.md]: oh right ok, will say I reported a bug instead

File: docs/team/candyhy.md (hunk 1 above)
Comment (candyhy) [reply docs/team/candyhy.md]: will correct my grammar!

**Review by w2vgd — APPROVED** [review]
LGTM

**Pull request merged**